### DISCUSSION TITLE
Cidades cobertas pela NFSe com PyNFe

### DISCUSSION THREAD

**Discussion by G3nilson** [TadaSoftware/PyNFe#356] [General]
Boa tarde, onde encontro a relação de cidades que é possível emitir a NFSe utilizando o PyNFe?

**Comment by EduardoFerreira22** [comment]
@leogregianin . Também gostaria de saber, pois estou com dúvidas se é possível emitir a NFSe em qualquer cidade ou necessitaria de integração com a prefeitura da cidade em que for feito a emissão?

Se for o caso, quais dados são necessários para essa integração?

**Comment by leogregianin** [comment]
@EduardoFerreira22 Não trabalhei com NFSe nesse projeto, me parece que está bem desatualizado e precisa passar por uma boa atualização de código.
